runtime/lua/vim/diagnostic.lua

```diff
@@ -1757,8 +1757,7 @@ local function render_virtual_lines(namespace, bufnr, diagnostics)
         string.rep(
           ' ',
           -- +1 because indexing starts at 0 in one API but at 1 in the other.
-          -- -1 for non-first lines, since the previous column was already drawn.
-          distance_between_cols(bufnr, diag.lnum, prev_col + 1, diag.col) - 1
```

Comment:
Originally a fix for the same issue as the issue my PR aims to fix.
 https://todo.sr.ht/~whynothugo/lsp_lines.nvim/13
https://git.sr.ht/~whynothugo/lsp_lines.nvim/commit/22ef87c

But clearly enough has changed since then that this isn't necessary.